openshift-ci/deploy_metallb.sh

```diff
@@ -38,6 +38,21 @@ spec:
   logLevel: debug
 EOF
 
+NAMESPACE="metallb-system"
+ATTEMPTS=0
+
+while [[ -z $(oc get endpoints -n $NAMESPACE webhook-service -o jsonpath="{.subsets[0].addresses}" 2>/dev/null) ]]; do
+  echo "still waiting for webhookservice endpoints"
+  sleep 10
+  (( ATTEMPTS++ ))
```

Comment:
why not putting an incremental wait?

```suggestion
  (( ATTEMPTS++ ))
  sleep $ATTEMPTS
```

Reply:
I'd say it's not worth the effort, in terms of cost of execution (10 seconds is already enough to wait), and unless we do some exponential backoff it will grow slower than we can appreciate the benefits of.

Reply:
ok